dts/arm/st/u5/stm32u595.dtsi

```diff
@@ -106,7 +106,8 @@
 			num-bidir-endpoints = <9>;
 			ram-size = <4096>;
 			maximum-speed = "high-speed";
-			clocks = <&rcc STM32_CLOCK(AHB2, 15U)>,
+			/* Enable OTG_HS PHY and peripheral clocks (OTGHSPHYEN | OTGEN) */
+			clocks = <&rcc STM32_CLOCK_BUS_AHB2 ((1 << 15) | (1 << 14))>,
```

Comment:
According to commit 5af9a0ae4c1f1f891feb9f9f85a0fd85b951c86f (merged before v4.1.0-rc2 tag), this is not needed.